### DISCUSSION TITLE
I don't wanna see it, please let me disable/hid it

### DISCUSSION THREAD

**Discussion by jmwielandt** [community/community#19775] [Other Feature Feedback, Questions, & Ideas]
This is my problem but probably shared with a lot of people. I don't want visual noise nor see stuff from people/projects I didn't explicitly followed, so please let us hid the tab "For you".
The fact two people are on the same GitHub organization doesn't mean they both known each other, or wanna see what the other one does.

Thanks.

**Comment by sasmlange** [comment]
You can click on **filter** and uncheck everything. This will not hide the tab, but it will always be empty. Not sure if this helps.
